test/functional/apps/context/classic/_context_navigation.ts

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
let's remove the whole file, it's not classic table specific navigating forth and back is tested